src/woo_publications/publications/admin.py

```diff
@@ -129,6 +129,33 @@ def remove_from_index(
     )
 
 
+@admin.action(
+    description=_(
+        "Recalculate the retention policy of the selected %(verbose_name_plural)s."
+    )
+)
+def recalculate_retention_policy(
```

Comment:
I think you missed the last comments from Marco :grimacing: 

```suggestion
@admin.action(
    description=_(
        "Reassess the retention policy of the selected %(verbose_name_plural)s."
    )
)
def reassess_retention_policy(
```